classifier-deep/apply/apply-labels/index.ts

```diff
@@ -54,11 +54,13 @@ class ApplyLabels extends Action {
 
 		if (manifestDbConnectionString) {
 			safeLog('has manifestDbConnectionString')
-			manifest = mongodb.MongoClient.connect(manifestDbConnectionString).then(async (db) => {
+			manifest = mongodb.MongoClient.connect(manifestDbConnectionString).then(async (client) => {
```

Comment:
I believe the new mongo client package is incompatible with our vscode-tools website's mongo version, I think we should just keep the old version. You can test with the connection string at https://ms.portal.azure.com/#@microsoft.onmicrosoft.com/resource/subscriptions/9b16b0df-65b6-46d3-b9c9-e4fbc86b1fa7/resourceGroups/vscode-tools/providers/Microsoft.DocumentDb/databaseAccounts/vscode-tools/mongoDbKeys

Reply:
Updated the connection string and tested.